### PR TITLE
adding client and server for websocket

### PR DESCRIPTION
**Pull request by RohanDudeja**

### CODE REVIEW

File: cmd/websocket/orders.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import (
+	"github.com/gorilla/websocket"
+	"log"
+	"os"
+)
+
+var done chan interface{}
+var interrupt chan os.Signal
+
+func main() {
```
Comment (mukesh0513) [inline cmd/websocket/orders.go]: why new main method for this?

File: cmd/websocket/orders.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"github.com/gorilla/websocket"
+	"log"
+	"os"
+)
+
+var done chan interface{}
```
Comment (mukesh0513) [inline cmd/websocket/orders.go]: why are these global variables?

File: internal/controller/websocket.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	"Tradeasy/internal/services/stock_exchange"
+	"encoding/json"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+	"log"
+	"time"
+)
+
+var upgrader = websocket.Upgrader{} // use default options
```
Comment (mukesh0513) [inline internal/controller/websocket.go]: do we need upgrader object to be initialised?

File: internal/controller/websocket.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+			}
+
+			if err := conn.WriteMessage(websocket.TextMessage, orderJson); err != nil {
+				log.Println("Error during writing stocks to websocket:", err)
+				return
+			}
+		}
+	}
+}
+func Home(c *gin.Context) {
```
Comment (mukesh0513) [inline internal/controller/websocket.go]: Purpose of this method?

**Comment by mukesh0513** [comment]
How will you handle if websocket connection broke in the middle?

**Comment by RohanDudeja** [comment]
updated files of this PR are used in PR #52

**Pull request closed without merging**